ami/models/hifigan_generator.py

```diff
@@ -0,0 +1,175 @@
+# Ref:
+# https://github.com/jik876/hifi-gan/blob/master/models.py
+
+import torch
+from torch.nn import Conv1d, ConvTranspose1d
+
```

Comment:
モジュールのimportに関してですが、`torch`の名前空間から全てアクセスするのは文字数が多くなり可読性が落ちます。また、自作のモジュールかpytorchのものかを見分けやすくするために、`import torch.nn as nn`はあると良いです。
`import torch.nn.functional as F`もあると`Leakey_relu`への参照が行いやすく良いかと。
```suggestion
import torch
import torch.nn as nn
import torch.nn.functional as F
```

Reply:
https://github.com/MLShukai/ami/commit/3c7f62578fd6f42d10123f4956ac2e8dc8049c0d で修正